autoprotocol/protocol.py

```diff
@@ -5046,85 +5046,109 @@ def provision(self, resource_id, dests, volumes):
           Resource ID from catalog.
         dests : Well or WellGroup or list(Well)
           Destination(s) for specified resource.
+        amounts : str or Unit or list(str) or list(Unit)
+          Mass(es) to transfer of the resource to each destination well.  If
+          one mass is specified, each destination well receive that mass of
+          the resource.  If destinations should receive different mass, each
+          one should be specified explicitly in a list matching the order of the
+          specified destinations.
+          Note: If value is provided for this field, value for 'volumes' must be skipped
         volumes : str or Unit or list(str) or list(Unit)
-          Volume(s) to transfer of the resource to each destination well.  If
-          one volume of specified, each destination well recieve that volume of
-          the resource.  If destinations should recieve different volumes, each
+          Volume to transfer of the resource to each destination well.  If
+          one volume is specified, each destination well receive that volume of the resource.
+          If destinations should receive different volumes, each
           one should be specified explicitly in a list matching the order of the
           specified destinations.
+          Note: If value is provided for this field, value for 'amounts' must be skipped
```

Comment:
nit: I'd rephrase to something like `volumes` and `amounts` arguments are mutually exclusive. Only one is required.

Reply:
Fixed